rulesdoc.md

```diff
@@ -1009,11 +1009,11 @@ You can maintain flight for a limited time. On your turn, you may take flight us
 - [Power Word: Lightning](#Power-Word-Lightning)
 - [Project Image](#Project-Image)
 
-##### Eigth
+##### Eighth
 - [Fiendfire](#Fiendfire)
 - [Invulnerability](#Invulnerability)
 - [Slow Time](#Slow-Time)
-- [Stablize Warren](#Stablize-Warren)
+- [Stabilize Warren](#Stabilize-Warren)
```

Comment:
Unsure if you'll have to update roll20 if the links/titles change

Reply:
yeah, but thats basically just running a script, I can check that in after.